app/forms/collection_form.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# Form for a Collection
+class CollectionForm < ApplicationForm
+  accepts_nested_attributes_for :related_links
+
+  def self.immutable_attributes
+    ['druid']
+  end
+
+  attribute :druid, :string
+  alias id druid
+
+  def persisted?
+    druid.present?
+  end
+
+  attribute :lock, :string
+
+  attribute :version, :integer, default: 1
+
+  attribute :title, :string
+  validates :title, presence: true
+
+  # The Collection description maps to the cocina abstract
+  attribute :abstract, :string
```

Comment:
The form is supposed to cleave to the UI, not the persistence model. Can this be called "description" (or whatever it is in the UI)?